src/components/SearchForm.tsx

```
@@ -0,0 +1,74 @@
+                ['destinationStation', destinationStation],
+                ['outboundDateTime', selectedDate.format('YYYY-MM-DDTHH:mm:ss.SSS[Z]')],
+                ['numberOfChildren', '0'],
+                ['numberOfAdults', '1']],
+            );
+        }
+    };
+
+    return (
+        <div id = { 'form' }>
+            <h1 id = "search-title">Find your next journey...</h1>
+            <main role = "main" aria-labelledby = "search-title">
```

Biome

[error] The elements with the following roles can be changed to the following elements:
<main>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)